x-pack/plugins/lens/public/state_management/lens_slice.ts

```diff
@@ -870,49 +862,13 @@ export const makeLensReducer = (storeDeps: LensStoreDeps) => {
         },
       };
     },
-    [updateStateFromSuggestion.type]: (
```

Comment:
ℹ️ With the refactoring, this method which introduced in a previous PR is not needed anymore